lib/mssql.js

```diff
@@ -124,6 +124,9 @@ MsSQL.prototype.executeSQL = function(sql, params, options, callback) {
   }
   var innerCB = function(err, data) {
     debug('Result: %j %j', err, data);
+    if (data && data[0] && data[0].affectedRows !== undefined && data[0].affectedRows !== null) {
```

Comment:
See https://github.com/strongloop/loopback-connector-mssql/pull/118/files#diff-abeffab7adabcb9df63c1db55fec2fb4R257, I think we may want to replace `data[0].affectedRows !== undefined && data[0].affectedRows !== null` with `typeof data[0].affectedRows === 'number'`.

To be more safe, I think we should also check whether this is an `UPDATE` statement, to prevent the situation when somebody has a model property called `affectedRows`, to ensure `Model.find()` does not mistakenly convert `data` array into `data[0]`.

I am also wondering, what's the difference between `data[0].affectedRows` and `data[0].count`? The latter is used by `getCountForAffectedRows` at https://github.com/strongloop/loopback-connector-mssql/pull/118/files#diff-abeffab7adabcb9df63c1db55fec2fb4R257